Instagram/Instagram/Global/DesignSystem/Font/IDSFont.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  IDSFont.swift
+//  Instagram
+//
+//  Created by 소연 on 2022/04/04.
+//
+
+import UIKit
+
+public enum IDSFont {
+    static let body1 = UIFont.systemFont(ofSize: 17, weight: .regular)
+    static let body2 = UIFont.systemFont(ofSize: 16, weight: .regular)
+    static let body3 = UIFont.systemFont(ofSize: 15, weight: .regular)
+    static let body4 = UIFont.systemFont(ofSize: 14, weight: .regular)
+    static let body5 = UIFont.systemFont(ofSize: 13, weight: .regular)
```

Comment:
그냥 FYI 하나 던지고 갈게!
이건 자라나라머리머리 하면서 알게된 사실인데 나중에 폰트가 너무 많아지면 관리하기가 힘들어져서 size를 받는 형태로 extension을 만들기도해.

애플이 기본제공하는 system font가 .systemFont(ofSize: 10) 이런 식으로 돼있어서 그 형식 맞춰주면 더 편할것 같아. 
.btnText(ofSize: 32) 이렇게!

```suggestion
// 만약에 폰트 이름이 `KCC-Kimhwanki` 라면
    class func btnText(ofSize: CGFloat) -> UIFont {
      return UIFont(name: "KCC-Kimhwanki", size: size)!
  }
```

Reply:
오 좋은데요?? 좋은 정보 감사합니당 💋